lms/static/js/dashboard/legacy.js

```diff
@@ -95,6 +96,20 @@
             });
         }
 
+        function toggleCourseActionsDropdown(event) {
+
+            // Toggle the visibility control for the selected element
+            var dashboard_index = $(this).data('dashboard-index');
+            var dropdown_selector = 'div#actions-dropdown-' + dashboard_index;
+
+            event.preventDefault();
+            var dropdown = $(dropdown_selector);
+            dropdown.toggleClass('is-visible');
+
+            // add BI event here
```

Comment:
that's an issue from me copy pasting an earlier event, I've removed the note here.